### PR TITLE
[stable/traefik] values.yaml support for extraVolumes and extraVolume…

### LINKED ISSUE #15300: [stable/traefik] add values for volumes and volumeMounts
**Is your feature request related to a problem? Please describe.**
I have a log scraper that needs access to the traefik access logs. I expose the access logs by writing them to a directory in the traefik container `/var/log/traefik/access.log`. I need to be able to expose those logs to the underlying host through a volume of `hostPath`

**Describe the solution you'd like**
I would like to be able to configure volumes and volumeMounts with the values.yaml like so
```yaml
---
# traefik.values.yaml

accessLogs:
  enabled: true
  filePath: /var/log/traefik/access.log

extraVolumes: |
  - name: accesslogs
    hostPath:
      path: /var/log/traefik
      type: DirectoryOrCreate

extraVolumeMounts: |
  - mountPath: /var/log/traefik
    name: accesslogs
```

**Describe alternatives you've considered**
Haven't thought of any at the moment, I believe this would be an extensible feature of the Traefik helm chart. 

**Additional context**

### PR DESCRIPTION
**Pull request by ghost**
values.yaml support for extraVolumes and extraVolumeMounts in stable/traefik chart

#### What this PR does / why we need it:
I would like to be able to configure volumes and volumeMounts with the values.yaml like so
```yaml
---
# traefik.values.yaml

accessLogs:
  enabled: true
  filePath: /var/log/traefik/access.log

extraVolumes:
  - name: accesslogs
    hostPath:
      path: /var/log/traefik
      type: DirectoryOrCreate

extraVolumeMounts:
  - mountPath: /var/log/traefik
    name: accesslogs
```

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #15300

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/traefik/values.yaml [hunk 1]
```diff
@@ -471,3 +471,6 @@ tracing:
 #   entryPoints: ["http", "https"]
 #   address: https://authserver.com/auth
 #   trustForwardHeader: true
+
+extraVolumes:
```
Comment (scottrigby) [inline stable/traefik/values.yaml]: These just need comments to help the user understand what this means, and a commented out example of how to use it.

**Review by scottrigby — CHANGES_REQUESTED** [review]
Hi thanks for this PR. I like this in general.

@naseemkullah has suggested something similar here https://github.com/helm/charts/issues/3011#issuecomment-451232038 and in this PR (as of now still open for discussion) https://github.com/helm/helm/pull/5110. For a chart like this that already defines volumes and mounts, I think this makes sense. 

See my note about adding useful comments for the end user, but otherwise I think this looks good.

**Comment by KadenLNelson** [comment]
@scottrigby would you mind reviewing again please?

**Comment by maorfr** [comment]
can you get the tests green? :)

**Comment by KadenLNelson** [comment]
@maorfr Sorry, the test should be passing now. Thank you.

**Review by scottrigby — APPROVED** [review]

**Pull request merged**